app/assets/javascripts/discourse/app/controllers/password-reset.js

```diff
@@ -93,7 +93,7 @@ export default Controller.extend(PasswordValidation, {
               DiscourseURL.redirectTo(result.redirect_to || "/");
             }
           } else {
-            if (result.errors && !result.errors.password) {
+            if (result.errors.security_keys) {
```

Comment:
The server response contains: `errors: { security_keys: [...], password: [...] }`

I think the original code was not expecting to have password when `security_keys` is present. Im not this is not breaking anything else but that was the simplest fix I could do for now.